shuup/admin/templates/shuup/admin/auth/login.jinja

```diff
@@ -41,18 +41,12 @@
 {% block extra_js %}
 <script>
 $(function () {
-    var fields = $("input");
-
-    fields.each(function() {
-        $(this).on("change", function() {
-            if ($(this).val().length > 0) {
-                $('.btn-primary').removeClass('disabled');
-            } else {
-                $('.btn-primary').addClass('disabled');
-            }
-        })
-    });
-
+    if(/iPad/i.test(navigator.userAgent) ) {
```

Comment:
IMO, not necessary

Reply:
Let's not do these. But please check what is the real issue here. If it is impossible to align the logo on all devices then maybe hide it in mobile.

Reply:
the issue is not the logo, but the entire form, there is super extra space on the top on mobile devices so users need to scroll to the it